<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="tvmatp2261m.ad.infosys.com" name="com.tests.May27TestSuite4" tests="4" failures="1" timestamp="1 Jun 2016 12:30:49 GMT" time="2660.576" errors="1">
  <testcase name="SubmitOrder_Brand_Search_Home" time="1094.409" classname="com.tests.May27TestSuite4">
    <error type="java.lang.RuntimeException" message="Error occured while closing webdriver sessionA session is either terminated or not started (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 4 milliseconds
Build info: version: &#039;2.51.0&#039;, revision: &#039;1af067d&#039;, time: &#039;2016-02-05 19:15:17&#039;
System info: host: &#039;tvmatp2261m.ad.infosys.com&#039;, ip: &#039;10.155.162.81&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.11&#039;, java.version: &#039;1.7.0_71&#039;
Session ID: 728e88f6-41b4-492d-95e5-8e58b0ed38f1
Driver info: io.appium.java_client.android.AndroidDriver
Capabilities [{automationName=Appium, platform=LINUX, app=/Users/jithya.joseph/Desktop/SyscoEndeca-debug.apk, javascriptEnabled=true, appActivity=, networkConnectionEnabled=true, desired={automationName=Appium, newCommandTimeout=120, app=/Users/jithya.joseph/Desktop/SyscoEndeca-debug.apk, platformVersion=6.0, autoWebview=false, deviceName=Sysco Galaxy S6, platformName=Android, appActivity=, appPackage=}, locationContextEnabled=false, appPackage=, newCommandTimeout=120, platformVersion=6.0.1, databaseEnabled=false, autoWebview=false, deviceName=1215fc6c214b2204, platformName=Android, webStorageEnabled=false, warnings={}, takesScreenshot=true}]">
      <![CDATA[java.lang.RuntimeException: Error occured while closing webdriver sessionA session is either terminated or not started (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 4 milliseconds
Build info: version: '2.51.0', revision: '1af067d', time: '2016-02-05 19:15:17'
System info: host: 'tvmatp2261m.ad.infosys.com', ip: '10.155.162.81', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11', java.version: '1.7.0_71'
Session ID: 728e88f6-41b4-492d-95e5-8e58b0ed38f1
Driver info: io.appium.java_client.android.AndroidDriver
Capabilities [{automationName=Appium, platform=LINUX, app=/Users/jithya.joseph/Desktop/SyscoEndeca-debug.apk, javascriptEnabled=true, appActivity=, networkConnectionEnabled=true, desired={automationName=Appium, newCommandTimeout=120, app=/Users/jithya.joseph/Desktop/SyscoEndeca-debug.apk, platformVersion=6.0, autoWebview=false, deviceName=Sysco Galaxy S6, platformName=Android, appActivity=, appPackage=}, locationContextEnabled=false, appPackage=, newCommandTimeout=120, platformVersion=6.0.1, databaseEnabled=false, autoWebview=false, deviceName=1215fc6c214b2204, platformName=Android, webStorageEnabled=false, warnings={}, takesScreenshot=true}]
	at zRfghY.s.b(WebDriverFactory.java:48)
	at com.iwaf.framework.testng.TestNGTestListener.closeTestSession(TestNGTestListener.java:458)
	at com.iwaf.framework.testng.TestNGTestListener.onTestFailure(TestNGTestListener.java:291)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1693)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1677)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:702)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
Caused by: org.openqa.selenium.remote.SessionNotFoundException: A session is either terminated or not started (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 4 milliseconds
Build info: version: '2.51.0', revision: '1af067d', time: '2016-02-05 19:15:17'
System info: host: 'tvmatp2261m.ad.infosys.com', ip: '10.155.162.81', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11', java.version: '1.7.0_71'
Session ID: 728e88f6-41b4-492d-95e5-8e58b0ed38f1
Driver info: io.appium.java_client.android.AndroidDriver
Capabilities [{automationName=Appium, platform=LINUX, app=/Users/jithya.joseph/Desktop/SyscoEndeca-debug.apk, javascriptEnabled=true, appActivity=, networkConnectionEnabled=true, desired={automationName=Appium, newCommandTimeout=120, app=/Users/jithya.joseph/Desktop/SyscoEndeca-debug.apk, platformVersion=6.0, autoWebview=false, deviceName=Sysco Galaxy S6, platformName=Android, appActivity=, appPackage=}, locationContextEnabled=false, appPackage=, newCommandTimeout=120, platformVersion=6.0.1, databaseEnabled=false, autoWebview=false, deviceName=1215fc6c214b2204, platformName=Android, webStorageEnabled=false, warnings={}, takesScreenshot=true}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:164)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:189)
	at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:526)
	at zRfghY.t.c(WebDriverInstanceGenerator.java:131)
	at zRfghY.s.b(WebDriverFactory.java:45)
	... 23 more
]]>
    </error>
  </testcase> <!-- SubmitOrder_Brand_Search_Home -->
  <testcase name="SubmitOrder_Categories_Search_Home" time="146.727" classname="com.tests.May27TestSuite4"/>
  <testcase name="SubmitOrder_Brand_Search_Home" time="1094.385" classname="com.tests.May27TestSuite4">
    <failure type="java.lang.AssertionError" message="myActiveOrder_SubmitOrder:Fail expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: myActiveOrder_SubmitOrder:Fail expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertFalse(Assert.java:63)
	at com.components.pages.MyActiveOrderPage.myActiveOrder_SubmitOrder(MyActiveOrderPage.java:195)
	at com.tests.May27TestSuite4.SubmitOrder_Brand_Search_Home(May27TestSuite4.java:157)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- SubmitOrder_Brand_Search_Home -->
  <testcase name="SubmitOrder_Categories_Home" time="325.055" classname="com.tests.May27TestSuite4"/>
</testsuite> <!-- com.tests.May27TestSuite4 -->
